.travis.yml

```diff
@@ -27,6 +27,7 @@ script:
     - mvn -s settings.xml clean install -P hdp22 -pl '!im-crawler' --quiet
     - sh install-scripts.sh local
     - sudo service bdre start
+    - sh bdre-scripts/bin/login-check.sh "http://localhost:28850" admin zaq1xsw2 "bdre content page"
```

Comment:
@wiprodev  wrong path provided to login-check.sh, it should be **bdre-scripts/local/bin/login-check.sh** instead.